mirage/serializers/build.js

```diff
@@ -1,13 +1,3 @@
-import { Serializer } from 'ember-cli-mirage';
+import V3Serializer from './v3';
 
-export default Serializer.extend({
-  serialize(object, request) {
-    const response = object.attrs;
-
-    if (object.commit) {
-      response.commit = this.serializerFor('commit').serialize(object.commit, request);
-    }
-
-    return response;
-  }
-});
+export default V3Serializer.extend({});
```

Comment:
I think Mirage will fall back to using the `Application` serializer if we delete this, right? Given that inherits from `v3`, I think this could be deleted.